snippets/views_embedded_ruby.cson

```diff
@@ -27,6 +27,34 @@
       <% end
     """
 
+'.text.haml > .line.ruby.haml > .source.ruby':
```

Comment:
I divide these snippet in 3 selectors group: Only erb, only haml, both erb and haml